CHANGES.rst

```diff
@@ -7,6 +7,27 @@ never too late to start, so let's start here and now.
 Unreleased
 ----------
 * New option ``session=None`` to provide the session used for making http requests.
+* Bug fix: inlined styles are no longer sorted alphabetically. This preserves the input
```

Comment:
Does this need a newline *before* so it renders to a bullet point list in reStructuredText?

Reply:
Thanks! not very familiar with rst so I threw some markdown syntax in there without thinking. I checked the preview - it should be fixed now.